Revisiting Rock Paper Scissors Assignment: revise Git instruction (#25792)
TheOdinProject/curriculum: foundations/javascript_basics/revisiting_rock_paper_scissors.md

You can see all of your current branches using `git branch` with no other arguments. The branch that you’re currently on will be indicated with an asterisk. If you want to change back to `main` from any other branch, you can do so just like changing to any other branch using `git checkout main`.

> Once you’re done working on your feature branch and ready to bring the commits that you’ve made on it to your main branch,  you need to perform what is known as a `merge`.

Reviewer: This sentence reads weird to me, in a few places. Here is a suggestion, feel free to update it. (also removed two spaces after a comma)
```suggestion
Once you are done working on your feature branch and are ready to bring the commits that you’ve made on it to your main branch, you will need to perform what is known as a `merge`.
```

   merge develop id: "merge to main"
</pre>

> Sometimes,the same lines in a file will have been changed by two different branches. When this happens, you will have a merge conflict when you try and merge those branches together. In order to finish merging the branches you will have to first resolve the conflict, which will be covered in a future lesson.

Reviewer: This is missing a space after the first comma.
```suggestion
Sometimes, the same lines in a file will have been changed by two different branches. When this happens, you will have a merge conflict when you try and merge those branches together. In order to finish merging the branches you will have to first resolve the conflict, which will be covered in a future lesson.
```

1. Make sure to publish the project on GitHub Pages and add a live preview link in the [project lesson](https://www.theodinproject.com/paths/foundations/courses/foundations/lessons/rock-paper-scissors).

</div>

### Additional Resources

This section contains helpful links to related content. It isn’t required, so consider it supplemental.

* Actively learn the Git workflows discussed in this lesson with this [interactive **Visual Git Cheatsheet**](https://ndpsoftware.com/git-cheatsheet.html#loc=index;) by Andrew Peterson. **It's okay to be unfamiliar** with the variety of commands you'll interact with. You'll learn about the ones you're unfamiliar with later in the curriculum.
* Make pushing your local commits to remote branches **easier** with the command `git push -u origin <branch>`. It automatically links the local branch you push with the remote one. [Read this educative.io article](https://www.educative.io/edpresso/what-is-the-git-push--u-remote-branch-name-command) by Talha Ashar and commit faster to a remote branch with a simple `git push` command.

> * Learn Git Branching by going through this [interactive visualizer](https://learngitbranching.js.org/) by Peter Cottle. You can learn the substantial commands about branching while seeing how the commands you write affect your visually presented branch tree.

Reviewer: While you are updating this section, can you change the `*` to a `-`, per our style guide.

https://github.com/TheOdinProject/curriculum/blob/main/LAYOUT_STYLE_GUIDE.md#unordered-lists

1. Make sure to publish the project on GitHub Pages and add a live preview link in the [project lesson](https://www.theodinproject.com/paths/foundations/courses/foundations/lessons/rock-paper-scissors).

</div>

### Additional Resources

This section contains helpful links to related content. It isn’t required, so consider it supplemental.

- Actively learn the Git workflows discussed in this lesson with this [interactive **Visual Git Cheatsheet**](https://ndpsoftware.com/git-cheatsheet.html#loc=index;) by Andrew Peterson. **It's okay to be unfamiliar** with the variety of commands you'll interact with. You'll learn about the ones you're unfamiliar with later in the curriculum.
- Make pushing your local commits to remote branches **easier** with the command `git push -u origin <branch>`. It automatically links the local branch you push with the remote one. [Read this educative.io article](https://www.educative.io/edpresso/what-is-the-git-push--u-remote-branch-name-command) by Talha Ashar and commit faster to a remote branch with a simple `git push` command.

> 

Reviewer: Thanks for making these changes so quickly! However, this extra line will need to be removed.

Author: I'm not near my computer at the moment. Can you add your suggestion in an in-line comment,so I can commit the change please?

> - Actively learn the Git workflows discussed in this lesson with this [interactive **Visual Git Cheatsheet**](https://ndpsoftware.com/git-cheatsheet.html#loc=index;) by Andrew Peterson. **It's okay to be unfamiliar** with the variety of commands you'll interact with. You'll learn about the ones you're unfamiliar with later in the curriculum.
> - Make pushing your local commits to remote branches **easier** with the command `git push -u origin <branch>`. It automatically links the local branch you push with the remote one. [Read this educative.io article](https://www.educative.io/edpresso/what-is-the-git-push--u-remote-branch-name-command) by Talha Ashar and commit faster to a remote branch with a simple `git push` command.
> 
> - Learn Git Branching by going through this [interactive visualizer](https://learngitbranching.js.org/) by Peter Cottle. You can learn the substantial commands about branching while seeing how the commands you write affect your visually presented branch tree.

Reviewer: Sorry I didn't think about making this suggestion earlier. It does make it easier! :)
```suggestion
- Actively learn the Git workflows discussed in this lesson with this [interactive **Visual Git Cheatsheet**](https://ndpsoftware.com/git-cheatsheet.html#loc=index;) by Andrew Peterson. **It's okay to be unfamiliar** with the variety of commands you'll interact with. You'll learn about the ones you're unfamiliar with later in the curriculum.
- Make pushing your local commits to remote branches **easier** with the command `git push -u origin <branch>`. It automatically links the local branch you push with the remote one. [Read this educative.io article](https://www.educative.io/edpresso/what-is-the-git-push--u-remote-branch-name-command) by Talha Ashar and commit faster to a remote branch with a simple `git push` command.
- Learn Git Branching by going through this [interactive visualizer](https://learngitbranching.js.org/) by Peter Cottle. You can learn the substantial commands about branching while seeing how the commands you write affect your visually presented branch tree.
```
